OKTA-544449: Adding support for Splunk Cloud edition (#3812)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/log-streaming/index.md

```json
{
  "edition": "aws",

>   "host": "acme.splunkcloud.com"

Reviewer: ```suggestion
  "host": "acme.splunkcloud.com",
```

| Property      | Description                                                  | DataType                                                       | Nullable | Unique | Readonly | MinLength | MaxLength |
| ------------- | ------------------------------------------------------------ | -------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- |

> | edition       | The edition of the Splunk Cloud instance. Can be set to one of the values: `aws`, `gcp` or `aws_govcloud`     | String                                    | FALSE    | FALSE   | FALSE     |      -     |     -      |

Reviewer: ```suggestion
| edition       | The edition of the Splunk Cloud instance. Supported values: `aws`, `gcp`, or `aws_govcloud`     | String                                    | FALSE    | FALSE   | FALSE     |      -     |     -      |
```
